README.md

```diff
@@ -41,11 +41,6 @@ To build the project:
 npm run build
 ```
 
-In dev mode, you can use:
-```bash
-npm run watch
```

Comment:
is watch not working or?

Reply:
It does work, but will people use it now that we run it concurrently with the dev server?